pandas/core/arrays/base.py

```diff
@@ -1511,6 +1511,13 @@ def __array_ufunc__(self, ufunc: np.ufunc, method: str, *inputs, **kwargs):
                 self, ufunc, method, *inputs, **kwargs
             )
 
+        if method == "reduce":
+            result = arraylike.dispatch_reduction_ufunc(
+                self, ufunc, method, *inputs, **kwargs
+            )
+            if result is not NotImplemented:
+                return result
+
```

Comment:
can you not hook into the default_array_ufunc?

Reply:
thats the idea eventually